File: site/code-snippets/WebRtc/removeParticipantFromSession/sample.rb
require 'bandwidth'

include Bandwidth
include Bandwidth::WebRtc

BW_USERNAME = ENV['BW_USERNAME']
BW_PASSWORD = ENV['BW_PASSWORD']
BW_ACCOUNT_ID = ENV['BW_ACCOUNT_ID']

bandwidth_client = Bandwidth::Client.new(
    web_rtc_basic_auth_user_name: BW_USERNAME,
    web_rtc_basic_auth_password: BW_PASSWORD
)

web_rtc_client = bandwidth_client.web_rtc_client.client

session_id = "1234-abcd"
participant_id = "4312-dbca"

begin
    #web_rtc_client.remove_participant_from_session(BW_ACCOUNT_ID, session_id, participant_id)
    web_rtc_client.remove_participant_from_session(BW_ACCOUNT_ID, participant_id, session_id)
    #NOTE: This is currently improperly defined
rescue APIException => e
    puts e.response_code
end

File: site/code-snippets/WebRtc/removeParticipantFromSession/Sample.java
import com.bandwidth.BandwidthClient;
import com.bandwidth.http.response.ApiResponse;

import java.util.concurrent.CompletableFuture;
import java.util.concurrent.ExecutionException;

public class Sample {
    public static final String USERNAME = System.getenv("BW_USERNAME");
    public static final String PASSWORD = System.getenv("BW_PASSWORD");
    public static final String ACCOUNT_ID = System.getenv("BW_ACCOUNT_ID");

    public static void main(String[] args) {
        String participantId = "568749d5-04d5-483d-adf5-deac7dd3d521";
        String sessionId = "75c21163-e110-41bc-bd76-1bb428ec85d5";

        BandwidthClient client = new BandwidthClient.Builder()
                .webRtcBasicAuthCredentials(USERNAME, PASSWORD)
                .build();

        try {
            CompletableFuture<ApiResponse<Void>> completableFuture = client.getWebRtcClient().getAPIController().removeParticipantFromSessionAsync(ACCOUNT_ID, participantId, sessionId);
            System.out.println(completableFuture.get().getResult());
        } catch (InterruptedException | ExecutionException e) {
            System.out.println(e.getMessage());
        }
    }
}